README.md

```diff
@@ -94,19 +94,20 @@ $ minikube addons disable metrics-server
 
 The following versions are supported and work as we test against these versions in their respective branches. But note that other versions might work!
 
-| kube-prometheus stack | Kubernetes 1.14 | Kubernetes 1.15 | Kubernetes 1.16 | Kubernetes 1.17 | Kubernetes 1.18 | Kubernetes 1.19 |
-|-----------------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|
-| `release-0.3`         | ✔               | ✔               | ✔               | ✔               | ✗               | ✗               |
-| `release-0.4`         | ✗               | ✗               | ✔ (v1.16.5+)    | ✔               | ✗               | ✗               |
-| `release-0.5`         | ✗               | ✗               | ✗               | ✗               | ✔               | ✗               |
-| `release-0.6`         | ✗               | ✗               | ✗               | ✗               | ✔               | ✔               |
-| `HEAD`                | ✗               | ✗               | ✗               | ✗               | x               | ✔               |
+| kube-prometheus stack | Kubernetes 1.14 | Kubernetes 1.15 | Kubernetes 1.16 | Kubernetes 1.17 | Kubernetes 1.18 | Kubernetes 1.19 | Kubernetes 1.20 |
+|-----------------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|
+| `release-0.3`         | ✔               | ✔               | ✔               | ✔               | ✗               | ✗               | ✗               |
```

Comment:
Should we keep supporting `release-0.3`?

Reply:
Lets remove the mention of release-0.3 yes, we will still support it. This is compatibility rather than support scheme.

I would stick to last 5 releases (including master) and last 5 k8s versions.

Reply:
Ok, I'll remove the 1.14 and 1.15 rows. However, since we don't have any version fully compatible with 1.16, is it meaningful to keep release-0.3 for now?

Reply:
> Lets remove the mention of release-0.3 yes, we will still support it. 
> I would stick to last 5 releases (including master) and last 5 k8s versions.

Yes as mentioned, I believe 5 versions master being one of them, removes the release-0.3 no?